.github/workflows/pr-reminder.yml

```diff
@@ -0,0 +1,22 @@
+name: PR Reminder
+
+on:
+  pull_request:
+    types:
+      - opened
+
+jobs:
+  pr-reminder:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
I would suggest that we use v4. (I don't know why it doesn't let me suggest the change)

Reply:
Actually, we don't need a checkout here at all :D I'm removing it altogether.